test/assets/testFiles/test1.js

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
These four files ought to have some sort of preamble to describe what they are for.